deploy/shakenfist_ci/tests/test_snapshots.py

```diff
@@ -36,15 +38,31 @@ def test_single_disk_snapshots(self):
         # Take a snapshot
         snap1 = self.test_client.snapshot_instance(inst1['uuid'])
         self.assertIsNotNone(snap1)
-        snapshots = self.test_client.get_instance_snapshots(inst1['uuid'])
+
```

Comment:
These repeated blocks are replaced by code in the client to handle async here when https://github.com/shakenfist/client-python/pull/70 merges (this one must merge first).